data/views/node.2.0.json

```diff
@@ -50,5 +50,19 @@
     <% } %>
     <% } %>
     "type": "<%=type%>",
-    "workflows": "<%=basepath %>/nodes/<%=id%>/workflows" 
+    "workflows": "<%=basepath %>/nodes/<%=id%>/workflows",
+    <% if (ibms === null) { %>
```

Comment:
A  minor suggestion, put the ibm together with obm will help people to compare the difference between them.